.github/workflows/deploy-staging-networks.yml

```diff
@@ -124,6 +124,11 @@ jobs:
           BOT_SWAPS_FOLLOW_CHAIN=PENDING
           BOT_SWAPS_TX_INTERVAL_SECONDS=350
 
+          RPC_INGRESS_ENABLED=true
+          RPC_INGRESS_HOST=staging.alpha-testnet.aztec-labs.com
+          RPC_INGRESS_STATIC_IP_NAME=staging-rc-1-ingress
```

Comment:
This A record is already setup and is currently unused. I'd say we reuse them rather than creating new ones